package.json

```diff
@@ -9,18 +9,19 @@
   "typings": "dist/index.d.ts",
   "scripts": {
     "clean": "rm -rf dist",
-    "compile": "babel src --out-dir dist -s --source-map --extensions '.ts,.tsx' --ignore src/**/__tests__,src/**/__stories__ && tsc --emitDeclarationOnly && rm dist/styled-system.d.*",
+    "compile": "babel src --out-dir dist -s --source-map --extensions '.ts,.tsx' --ignore src/**/__tests__,src/**/__stories__",
     "lint": "tslint -c tslint.json --project tsconfig.json",
     "precommit": "lint-staged",
     "prepush": "yarn run type-check",
-    "prepublishOnly": "yarn clean && yarn compile",
+    "prepublishOnly": "yarn clean && yarn compile && yarn type-declarations",
     "prettier": "prettier",
     "prettier-project": "yarn prettier-write 'src/**/*.{ts,tsx}'",
     "prettier-write": "yarn prettier --write",
     "semantic-release": "semantic-release",
     "test": "yarn type-check && yarn jest",
     "type-check": "tsc --noEmit --pretty",
-    "watch": "yarn compile -w"
+    "type-declarations": "tsc --emitDeclarationOnly && rm dist/styled-system.d.*",
```

Comment:
@alloy - i imagine this `styled-system.d` is here because we still need to move to `@types` and once there this can go?